Optimise Getstarted - Index (#536)
opengisch/QField-docs: documentation/get-started/index.en.md

    Download the latest <a href="https://qfield.org/get_latest/?platform=macos" target="_blank">MacOS
    version</a> (Universal).

> QField is available for both Android and all IoS devices (the newer versions) and can also be accessed on the Desktop (Windows only) through an application that is available on Microsoft Store.

Reviewer: ```suggestion
QField is available for Android and iOS devices and can also be accessed on the Desktop (Windows, macOS and Linux).
```

Use QField for optimised data collection outside in any place on earth. 
Its optimized user interface hides the full power of [QGIS](https://qgis.org) under the hood.

## QFieldCloud

> QFieldCloud is one and the recommended option to seamlessly integrate your team's field work. 

Reviewer: ```suggestion
[QFieldCloud](https://qfield.cloud) allows you to seamlessly integrate your team's field work. 
```

> It is a cloud-based Service and with its unique technology it will allow your team to focus on what's important, making sure you efficiently get the best field data possible.

Reviewer: ```suggestion
It is a cloud-based service and with its unique technology it will allow your team to focus on what's important, making sure you efficiently get the best field data possible.
```

    Download the latest <a href="https://qfield.org/get_latest/?platform=macos" target="_blank">MacOS
    version</a> (Universal).

QField is available for Android and iOS devices and can also be accessed on the Desktop (Windows, macOS and Linux).
Use QField for optimised data collection outside in any place on earth.

> Its optimized user interface hides the full power of [QGIS](https://qgis.org) under the hood.

Reviewer 2: ```suggestion
Its optimized user interface exposes through a an easy-to-use front-end that gets out of the way the full power of [QGIS](https://qgis.org) under the hood.
```

Reviewer 2: Rational: I feel "hiding" is a bit negative. We don't hide features, we channel through through a much cleaner interface that's friendlier and won't overwhelm you on first approach.

QField is available for Android and iOS devices and can also be accessed on the Desktop (Windows, macOS and Linux).
Use QField for optimised data collection outside in any place on earth.

> Its optimized user interface exposes through a an easy-to-use front-end that gets out of the way the full power of [QGIS](https://qgis.org) under the hood.

Reviewer: There's a typo in here and I think it's a bit too verbose

```suggestion
Its optimized user interface exposes the full power of [QGIS](https://qgis.org) through an easy-to-use front-end.
```
